<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UPDATE</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image/srcc.qrc">
    <normaloff>:/img/logo1.jpg</normaloff>:/img/logo1.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: qconicalgradient(cx:0.548913, cy:1, angle:0, stop:0 rgba(0, 0, 103, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QLabel" name="label_invent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>201</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/update.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>UPDATE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Qty_Sold">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="quantity_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Qty:_Sold:</string>
   </property>
  </widget>
  <widget class="QLabel" name="product_name">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product Name :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Product_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="amount_sold">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Amount_Sold:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Amt_edit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image/srcc.qrc"/>
 </resources>
 <connections/>
</ui>
